[2023-08-14T19:13:04.207+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ny_taxi_Pipeline.s3_file_check manual__2023-08-14T19:11:57+00:00 [queued]>
[2023-08-14T19:13:04.343+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ny_taxi_Pipeline.s3_file_check manual__2023-08-14T19:11:57+00:00 [queued]>
[2023-08-14T19:13:04.343+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 31
[2023-08-14T19:13:04.428+0000] {taskinstance.py:1327} INFO - Executing <Task(S3KeySensor): s3_file_check> on 2023-08-14 19:11:57+00:00
[2023-08-14T19:13:04.544+0000] {standard_task_runner.py:57} INFO - Started process 15412 to run task
[2023-08-14T19:13:04.608+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ny_taxi_Pipeline', 's3_file_check', 'manual__2023-08-14T19:11:57+00:00', '--job-id', '1635', '--raw', '--subdir', 'DAGS_FOLDER/ny_taxi_datapipeline.py', '--cfg-path', '/tmp/tmp7j6x839i']
[2023-08-14T19:13:04.649+0000] {standard_task_runner.py:85} INFO - Job 1635: Subtask s3_file_check
[2023-08-14T19:13:04.787+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-08-14T19:13:05.100+0000] {task_command.py:410} INFO - Running <TaskInstance: ny_taxi_Pipeline.s3_file_check manual__2023-08-14T19:11:57+00:00 [running]> on host d4028c5ba2ac
[2023-08-14T19:13:05.902+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ny_taxi_Pipeline' AIRFLOW_CTX_TASK_ID='s3_file_check' AIRFLOW_CTX_EXECUTION_DATE='2023-08-14T19:11:57+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-14T19:11:57+00:00'
[2023-08-14T19:13:05.940+0000] {s3.py:103} INFO - Poking for key : s3://myawsbucket19080/hiveStep/raw/tripdata.csv
[2023-08-14T19:13:05.979+0000] {base.py:73} INFO - Using connection ID 'aws_connection' for task execution.
[2023-08-14T19:13:05.980+0000] {connection_wrapper.py:340} INFO - AWS Connection (conn_id='aws_connection', conn_type='aws') credentials retrieved from login and password.
[2023-08-14T19:13:21.160+0000] {s3.py:103} INFO - Poking for key : s3://myawsbucket19080/hiveStep/raw/tripdata.csv
[2023-08-14T19:13:31.351+0000] {s3.py:103} INFO - Poking for key : s3://myawsbucket19080/hiveStep/raw/tripdata.csv
[2023-08-14T19:13:41.513+0000] {s3.py:103} INFO - Poking for key : s3://myawsbucket19080/hiveStep/raw/tripdata.csv
[2023-08-14T19:13:41.683+0000] {base.py:255} INFO - Success criteria met. Exiting.
[2023-08-14T19:13:41.832+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=ny_taxi_Pipeline, task_id=s3_file_check, execution_date=20230814T191157, start_date=20230814T191304, end_date=20230814T191341
[2023-08-14T19:13:42.148+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-14T19:13:43.285+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
